messages/root/messages.go

```diff
@@ -7,6 +7,8 @@ var (
 	RootDoNotUpdate = "Do not receive update notification"
 	RootLogDebug    = "Displays log at a debug level"
 	RootLogLevel    = "Set the logging level, \"debug\", \"info\", or \"error\"."
+	RootFlagOut     = "Exports the output to the given <file_path/file_name.ext>"
+	RootFlagFormat  = "Changes the output format passing the json value to the flag"
```

Comment:
Changes the output format passing 'json, toml or yaml' to the flag
@gabriel-azion is this msg okay?

Reply:
Yesss, ok